### PR TITLE
New secret exchange flow using sessionStorage + iframe + postMessage

### PR DESCRIPTION
**Pull request by matheus23**
Depends on https://github.com/fission-suite/webnative/pull/220

The iframe itself is super light and doesn't have huge dependencies (no IPFS).

I've also removed the "Warming up" UI, but only when the auth lobby uses the new exchange flow.

This PR can be tested on production at:
* https://philipps-auth-test.fission.app/ The lobby part
* https://dashboard-auth-test.fission.app/ An app with the new webnative change to support this

### CODE REVIEW

**Review by icidasset — APPROVED** [review]

**Comment by icidasset** [comment]
By the way, not entirely sure about removing the warming up thing. We still need to show more specific loading messages when it needs to: load filesystem + create directories & files for permissions. But we can do that later 👌

**Comment by matheus23** [comment]
> By the way, not entirely sure about removing the warming up thing. We still need to show more specific loading messages when it needs to: load filesystem + create directories & files for permissions. But we can do that later :ok_hand:

Oh right, we can't get away with no filesystem changes, as the files/directories need to be initialized such that we have some read keys for them.

My thought process for removing the warmup thing was "it's faster now, so we can switch back to a simpler loading screen -> higher perceived performance for users".

But yeah maybe that's a discussion for another time. I'm thinking of removing that change for now :thinking:

File: src/Application/External/Context.elm [hunk 1]
```diff
@@ -356,3 +368,7 @@ confirmPaths =
 
 newFlowSdkVersion =
     Semver.version 0 22 0 [] []
+
+
+sessionStorageSdkVersion =
+    Semver.version 0 24 2 [] []
```
Comment (icidasset) [inline src/Application/External/Context.elm:374]: Lets make this 0.24.0, we can get it in the next release 👀

**Pull request merged**